tests/sdl/whitelists.py

```diff
@@ -92,10 +92,8 @@ class OvmsBaseImageType(Enum):
 }
 
 whitelisted_libraries = {
-    OvmsImageType.UBUNTU20: {"default": libraries[OvmsBaseImageType.COMMON] | libraries[OvmsBaseImageType.UBUNTU],
-                             "python": libraries[OvmsBaseImageType.UBUNTU20_PYTHON]},
```

Comment:
please also remove libraries[OvmsBaseImageType.UBUNTU20_PYTHON] and libraries[OvmsBaseImageType.UBUNTU22_PYTHON]

Reply:
done